Siminar7/controller.py

```diff
@@ -0,0 +1,69 @@
+from statistics import mode
+from interface import start_input
+from interface import start2_input
+from interface import add_input
+from interface import edit_input
+from interface import del_input
+from interface import viewer
+from generator import get_ide
+from generator import get_name
+from generator import get_birthday
+from generator import get_work
+from generator import get_telephone
+from generator import gener_collection
+from logger import log_collection
+from logger import save
+from logger import load
+from edit import add
+from edit import edit
+from edit import delit
+
+
+global ide
+global name
+global birthday
+global work
+global telephone
+
+def controller():
+    ide, name, birthday, work, telephone, = [], [], [], [], [] 
+    #для просмотра пустых списков, до загрузки и генерации
+
+    while 1:
```

Comment:
```suggestion
    while True:
```
Обычно так вечный цикл определяют.